### PR TITLE
feat: add feature login

### PR DESCRIPTION
**Pull request by khajackie2206**
This is a demo of login feature

### CODE REVIEW

File: src/Controller/ProfileController.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?php
```
Comment (nhiv0) [inline src/Controller/ProfileController.php:1]: i think Profile is not enough to be a Controller

File: src/DataFixtures/AppFixtures.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+namespace App\DataFixtures;
+
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Persistence\ObjectManager;
+
+class AppFixtures extends Fixture
+{
+    public function load(ObjectManager $manager): void
+    {
+        // $product = new Product();
```
Comment (nhiv0) [inline src/DataFixtures/AppFixtures.php]: should remove unused code

File: src/Controller/ProfileController.php (hunk 1 above)
Comment (khajackie2206) [reply src/Controller/ProfileController.php:1]: oke I will fix it, thank u so much <3

File: src/DataFixtures/AppFixtures.php (hunk 2 above)
Comment (khajackie2206) [reply src/DataFixtures/AppFixtures.php]: oke I will remove it

**Pull request closed without merging**